### PR TITLE
PEP8 related changes

### PR DESCRIPTION
**Pull request by deathowl**
Also added feature to add new alias to a PAD.

### CODE REVIEW

**Review by yegorshr — DISMISSED** [review]
Thank you for this comment

File: params.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
-Username		= None 			# CDNetworks Portal username
-Password		= None			# CDNetworks Portal password
-svcGroupName	= None			# CDNetwork contol group
-srcPADName		= None			# Example www.mydomain.com, To copy from full PAD configuration or specific SAM rule you should use service with type 0.
-destPADName		= None			# Example www.myNewdomain.com, New PAD or existing one for cloning specific SAM rule.
-								# In case of SAM rule cloning sourse and target PAD can be the same.
-origin			= None			# Specify origin in case it is different from sourse, if you line to avoid prompt input set as '#DONTASKUSER#'
-description		= None			# New PAD description, if you line to avoid prompt input set as '#DONTASKUSER#'
\ No newline at end of file
+username = None
```
Comment (yegorshr) [inline params.py:1]: Please keep comments. Otherwise I don't think that it is clear to people what to use here.

File: cdnetworks/actions.py [hunk 2]
```diff
@@ -4,11 +4,7 @@ def __init__(self, api_base, session_token, api_key):
         self.session_token = session_token
         self.api_key = api_key
 
-    def create_pad(self, contract_number, new_pad_name, origin, description=None):
```
Comment (yegorshr) [inline cdnetworks/actions.py]: Why did you remove create_pad function?

**Review by yegorshr — CHANGES_REQUESTED** [review]
Thank you for contributing.
2 comments added

File: cdnetworks/actions.py (hunk 2 above)
Comment (deathowl) [reply cdnetworks/actions.py]: it wasn't used by anywhere in the project
